content/maxwell1_fundamentals/physical_principles/maxwells_equations_in_homogeneous_media/index.rst

```diff
@@ -3,7 +3,9 @@
 Maxwell's Equations in Homogeneous Media
 ========================================
 
-.. todo:: overview of second order maxwell system? 
+.. todo:: under construction!
+
+In this section, we combine the first-order partial :ref:`differential equations <differential_equations_time>` into second-order equations for :math:`\mathbf{e}` or :math:`\mathbf{h}` and show how EM fields propagate in homogenous media. The plane wave solutions have propogation and attenuating components and the relative balance of these is determined by the complex wave number. When electrical conductivity :math:`\sigma` is zero, the electromagnetic waves propagate without attenuation. When :math:`\sigma` is large, as it is with most earth rocks, the EM waves are diffusive and attenuate rapidly within one or two wavelengths. In many problems, the wave propagation portion of the equation (or effectively, the displacement current), can be neglected and quasi-static Maxwell's equations can be solved. The following content parallels that offered in many EM resources :cite:`ward1988,griffiths1999,stratton1941`.
```

Comment:
before merging on to master, can we split large paragraphs into lines (it makes version controlling more transparent). A text editor should be able to do this fairly easily (ie. if you use sublime 'Edit--> Wrap' ). If you run into issues, I can do it and push the changes before we merge this pull request